lib/parallel_tests/cli.rb

```diff
@@ -13,6 +13,8 @@ def run(argv)
       num_processes = ParallelTests.determine_number_of_processes(options[:count])
       num_processes = num_processes * (options[:multiply] || 1)
 
+      options[:use_test_env_number_for_first_process] ||= ParallelTests.always_use_test_env_number_for_first_process?
```

Comment:
It's `||=` to allow for the option to be set on the command line as well.

I like having the check code in `ParallelTests` because it's a similar construction to `ParallelTests.determine_number_of_processes`. I can change it if you prefer, though.